src/components/RestaurantButton.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+function RestaurantButton(props) {
+  return (
+    <button className="btn btn-primary" onClick={props.customerOrders}>
+      Add
```

Comment:
Nice one! I think it would be more intuitive to call the prop with the actual function name - `orderOne` instead of `customerOrders`.